storages/memory-storage/tests/memory_storage.rs

```diff
@@ -1,11 +1,13 @@
+use async_trait::async_trait;
 use {gluesql_core::prelude::Glue, gluesql_memory_storage::MemoryStorage, test_suite::*};
```

Comment:
Please group `use` statements

```suggestion
use {gluesql_core::prelude::Glue, gluesql_memory_storage::MemoryStorage, test_suite::*}, async_trait::async_trait;
```